src/components/ui/FileInput/FileInput.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import FileInput from './FileInput';
+
+export default {
+  title: 'Ui/FileInput',
+  component: FileInput,
+  argTypes: {},
+} as ComponentMeta<typeof FileInput>;
+
+const Template: ComponentStory<typeof FileInput> = (args) => (
+    <FileInput {...args}/>
```

Comment:
I think formatting is not good here, too many spaces
Why prettier didn't fix it in lint-staged?